WebClient/webpack.config.js

```diff
@@ -0,0 +1,77 @@
+const path = require('path');
+/* jshint unused:false */
+const webpack = require('webpack');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = env => {
+    const devMode = !env || !env.production;
+
+    return {
+        entry: './src/app.js',
+        output: {
+            filename: 'assets/js/[chunkhash].[name].bundle.js',
+            path: path.resolve(__dirname, 'dist')
+        },
+        module: {
+            rules: [
+                // html-loader
+                { test: /\.html$/, use: ['html-loader'] },
+                // css-loader
+                { 
+                    test: /\.css$/, 
+                    use: [
+                        MiniCssExtractPlugin.loader,
+                        {
+                            loader: 'css-loader',
+                            options: {
+                            sourceMap: devMode,
+                            url: false,
+                            import: false
+                            }
+                        }                    
+                    ]
+                },
+                // file-loader
+                {
+                    test: /\.(jpg|png|gif|svg)$/,
+                    use: [
+                        {
+                            loader: 'file-loader',
+                            options: {
+                                name: '[name].[ext]',
+                                outputPath: './assets/media/'
```

Comment:
Reminder: could be necessary to use a relative path or a definable base path

Reply:
In this pull request the path is relative. I've also changed the config file in the 'details' branch and made math relative there as well.